ESMCI/cime
# Jgfouca/branch for acme split 2019 02 28

Apologies for the big PR, I will annotate.

Change list:
* create_test --single-exe capability
* bless_test_results upgrades (use Case instead of xmlquery, autodetect newest test batch)
* change test-suite definition format
* add kokkos as a buildable submodule of e3sm
* make sure to pick up all doctests throughout scripts/lib

Test suite: scripts_regression_tests.py
Test baseline: 
Test namelist changes: 
Test status: [bit for bit, roundoff, climate changing]

Fixes [CIME Github issue #]

User interface changes?: 

Update gh-pages html (Y/N)?:

Code review: @jedwards4b
